Makefile

```diff
@@ -75,13 +75,16 @@ tetragon-bpf-container:
 verify: tetragon-bpf
 	sudo contrib/verify/verify.sh bpf/objs
 
-.PHONY: tetragon tetra tetragon-operator tetragon-alignchecker
+.PHONY: tetragon tetra tetragon-operator tetragon-alignchecker bench
```

Comment:
How about we name the binary `tetragon-bench`?

Reply:
Could we also add the binary to `.gitignore`?

Reply:
yes, good idea, thanks